### PR TITLE
New version: chaiNNer-org.chaiNNer version 0.20.2

### PR DESCRIPTION
**Pull request by onyx-nxt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122830)

### CODE REVIEW

**Comment by KK-Designs** [comment]
> Hello @KK-Designs,
> 
> During installation testing, this application failed to install without user input. Did you forget to add Silent or SilentWithProgress switches?
> 
> This can also happen when a dependency is missing. You can test with https://github.com/microsoft/winget-pkgs#test-your-manifest.
> 
> Please investigate a fix and resubmit the Pull Request.
> 
> You may also try using the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) to determine the proper installer type. If the installer type is MSIX, MSI, or a known installer technology like NullSoft, Inno, etc. the wingetcreate tool can detect them and the winget client will know what switches to pass. If it's a .exe installer of an unknown type, you will need to search to determine the proper switches for Silent and SilentWithProgress.
> 
> Template: msftbot/validationError/installation/unattended

Works fine for me, it does make windows ask for a firewall thing tho, how would I fix this?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Clicking "Open Setup Log" yields nothing.

File: manifests/c/chaiNNer-org/chaiNNer/0.20.2/chaiNNer-org.chaiNNer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: chaiNNer-org.chaiNNer
+PackageVersion: 0.20.2
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /SilentWithProgress
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/c/chaiNNer-org/chaiNNer/0.20.2/chaiNNer-org.chaiNNer.installer.yaml:11]:
```suggestion
- Architecture: x64
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**